doc/source/quickstart.rst

```diff
@@ -0,0 +1,49 @@
+Quick Start
+===========
+
+Prerequisites
+-------------
+
+Kiwi NG should be also installed on your system:
+
+.. code-block:: shell
+    :caption: Installing Kiwi NG from pip
+
+    $ pip install kiwi
+
+Also you need an example content description, so clone the official Kiwi NG image examples:
+
+.. code-block:: shell
+    :caption: Getting an example content description:
+
+    $ git clone https://github.com/OSInside/kiwi
+
+Build Image
+-----------
+
+To create your initial image, which serves as a basic system disk image compatible with full virtualization systems such as QEMU, execute the following command:
+
+.. code-block:: shell
+    :caption: Building Image
+
+    $ echo TBD
```

Comment:
berrymill -i TBD build --target-dir /tmp/tbd

Reply:
--debug for more detailed output

Reply:
--clean so you dont have to manually clean the target-dir when rebuilding

Reply:
--cpu for when the boxbuild failes with https://github.com/isbm/berrymill/issues/35

Reply:
--local for not using a Box env